### ISSUE TITLE
BUG: Multi-Selection Drag issue with dragMode: absolute

### ISSUE THREAD

**Issue by damnslow** [artf/grapesjs#3770]
### GrapesJS version

- [X] I confirm to use the latest version of GrapesJS

### What browser are you using?

Chrome v93

### Reproducible demo link

https://codepen.io/damnslow/pen/yLXXOxQ

### Describe the bug

**How to reproduce the bug?**
1. Select both the elements
2. Drag using drag icon

**What is the expected behavior?**
Both elements should move together

**What is the current behavior?**
Only one element moves.

If is necessary to execute some code in order to reproduce the bug, paste it here below:
No.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by damnslow** [comment]
Hi @artf, is there any workaround to solve this?

**Comment by artf** [comment]
Absolute positioning, right now, has more than one issue and I'd like to address them all here:
- [ ] Wrong position on a different Devices
- [ ] Wrong position on a zoomed/panned canvas
- [ ] Support touch devices

I don't know exactly when I'll start working on it but I hope soon, and as always, any help is appreciated.

**Comment by ahmet1340** [comment]
Did you find time to deal with it? I'm currently working on a drag and drop project and I'm having an issue with mispositioning on different devices also I can't use the setzoom function for mobile. I would be very grateful if you would take care of this issue. (Google translate used.)

**Comment by Adham380** [comment]
These issues have still not been resolved and are frankly annoying. Changing the absolute dragging to a % instead of a px offset makes so much more sense and solves most issues, but for some reason wasn't chosen. Is there any plans to change this?

**Comment by artf** [comment]
@Adham380 PRs are welcome

**Comment by dalmatele** [comment]
Agree. It also affects the menu of NavBar in Mobile mode. If you set dragMode to "absolute" and move items in NavBar then switch to Mobile Preview mode, you'll see NavBar does not work.

**Comment by vizardkill** [comment]
up

**Comment by voleep** [comment]
Up

**Comment by fmfeit** [comment]
Hi @artf,
this feature is crucial for my current project, so I have two questions:
1. Is it possible to pay you (or someone else) to prioritize this feature?
2. If not, can you outline the steps necessary to resolve the issue and give me a starting point?

**Comment by artf** [comment]
@fmfeit here you can find the related source of the [command](https://github.com/GrapesJS/grapesjs/blob/dev/src/commands/view/ComponentDrag.ts).

**Comment by itscarlosrufo** [comment]
Hi everyone,

Thanks a lot for the feedback and detailed discussion.

We’re happy to share that we’ve addressed some of the requested functionality. Also, we’ve introduced a new absolute mode plugin that should cover most of the use cases mentioned above.

You can read more about it in our new documentation here:
- https://app.grapesjs.com/docs-sdk/plugins/canvas/absolute-mode

Feel free to try it out and let us know if you have any further suggestions!

**Comment by asterisx** [comment]
Everything works great, except that on a zoomed canvas with absolute position, the drags are haywire, I think the movements are not compensated for the zoom level. It would be great if someone could share some pointers. I have tried a couple of approaches

1. Custom drag overriding the "core:component-drag", this did not work, the compensations are ignored.
2. Some others which was me just trying to hack my way to this (overriding setStyle and other messy solutions) 

None seem to work. This is the only thing stopping us from releasing our product so any help is greatly appreciated.

**Comment by tomekczm** [comment]
How is drag mode absolute supposed to work, when any selector is selected in selector manager.
For example if i select :hover and then move the object, it does not move and the css only seems to apply after i refresh it either through selecting some other component and then selecting the changed component or changing selector and then switching back.

When the :hover selector's position is changed it messes up editing when no selector is selected.
I think there should be a way just to disable dragging at any other state than the default, or a mode where dragging always applies the css to base class
